lib/linalg/BasisReader.h

```diff
@@ -58,92 +58,54 @@ class BasisReader {
      * @brief Returns true if the basis vectors at requested time are
      * different from the last requested basis vectors.
      *
-     * @pre 0 < numTimeIntervals()
-     * @pre 0 <= time
-     *
-     * @param[in] time Time at which we are interested in the basis vectors.
-     *
      * @return True if the basis vectors at the requested time are different
      *         from the last requested basis vectors.
      */
     bool
-    isNewBasis(
-        double time)
+    isNewBasis()
```

Comment:
Should this function and `d_last_basis_idx` be removed?

Reply:
I know this function is not used any more within libROM, but it wouldn't hurt to leave this function. I think the names can be confusing, so we can rename them, maybe `isFileLoaded()` and `bool d_file_loaded` ?

Reply:
This makes sense and is reflected to PR.